README.md

```diff
@@ -11,6 +13,46 @@ npm install --save analytics-client
 
 ## Usage
 
+Tracking a page view. The following reports an event named `[EFP] Page View` with default properties.
+
+```typescript
+import { createClient, WebTracker, createWebTracker } from 'analytics-client';
+
+const client = createClient({
+    endpoint: 'data.balena-cloud.com', // use 'data.balena-staging.com' for testing the integration
+    projectName: 'balena-project', // unique identifier that analytics-backend expects
+    componentName: 'etcher-featured-project', // short unique name of component
+    componentVersion: '0.52', // automated version reporting of component is beneficial
+
+});
+
+createWebTracker(client, 'EFP') // 2nd parameter defines the prefix of event names
+    .trackPageView();
+```
+
+Here is an example how a product like CLI would identify a user and report a login event (`[CLI] Login`) with certain event properties related to the login action.
+
+```typescript
+import { createClient } from 'analytics-client';
+
+const client = createClient({
+    endpoint: 'data.balena-cloud.com', // use 'data.balena-staging.com' for testing the integration
+    projectName: 'balena-project', // unique identifier that analytics-backend expects
+    componentName: 'CLI', // short unique name of component
```

Comment:
I'd probably avoid this an example tbh, `CLI` isn't what I'd normally think of as particularly unique (and if we were to do it today I'm sure we'd use eg `balena-cli`

Reply:
As discovered at the end of [this](https://www.flowdock.com/app/rulemotion/i-cli/threads/77cAobr10SuHLdgJPHntqRIMrn7) thread, CLI is currently not compatible with the `analytics-client` as it lacks support for the node environment.

Can we rename CLI to something else and recycle the example?

Reply:
Good point. I'll rewrite the example here with `balena-cli`. I guess the repository name of a component is a good candidate to be `componentName`.